src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -722,10 +723,7 @@ function MoneyRequestReportPreviewContent({
                                                 styles.mtn1,
                                             ]}
                                         >
-                                            <ActivityIndicator
-                                                color={theme.spinner}
-                                                size={40}
-                                            />
+                                            <ActivityIndicator size={40} />
```

Comment:
Let's use `CONST.ACTIVITY_INDICATOR_SIZE.LARGE` here for consistency

Reply:
Large activity indicator has size 36 so we cannot use it here

Reply:
I mean 40 seems like a mistake to me, but either works. I won't block on this 😄